core/templates/dev/head/components/question-directives/questions-list/questions-list.directive.ts

```diff
@@ -167,7 +167,7 @@ oppia.directive('questionsList', [
 
           ctrl.createQuestion = function() {
             if (ctrl.getSkill()) {
-              ctrl.initializeNewQuestionCreation();
+              ctrl.initializeNewQuestionCreation([ctrl.getSkill()]);
```

Comment:
Is the getSkill() value set when this is called from the topic editor? I just checked and I don't think it is. Can you check whether create question works without issues in the topic editor?

Reply:
As discussed offline, I am passing skill ids to the newQuestionCreation function after questions are selected in the topic editor..in the skill editor it happens as soon as the question creation dialog is opened